Fixed incorrect/confusing questions (#3254)
enkidevs/curriculum: javascript/javascript-core/functions-i/setting-default-values-with-short-circuiting.md

## Practice

> Complete both functions so `shortCirA` and `shortCirB` return `'John Doe'` and `'Jane Doe'` respectively on Undefined. If `name` is defined, then the function should just return the name:

Reviewer: ```suggestion
Complete both functions so `shortCirA` and `shortCirB` return `'John Doe'` and `'Jane Doe'`, respectively. 

If `name` is defined, then each function should return the name:
```

Author: It's unclear that John doe and Jane doe should be returned on undefined

Author: See my new version

## Practice

Complete both functions. 

When `name` is undefined, `shortCirA` and `shortCirB` should return `'John Doe'` and `'Jane Doe'` respectively. 

If `name` is defined, then each function should return the name:

```javascript
function shortCirA(name){
  return ??? || 'John Doe';
}
function shortCirB(name){
  return name ??? 'Jane Doe';

> }

Reviewer: What do you think of making the question testing both old (via ||) and new (via ??) way of short circuiting:

````md
Complete both functions:

1. When `name` is any falsy value, `shortCirA` should return `'John Doe'`.

2. When `name` is `undefined` or `null`, `shortCirB` should return `'Jane Doe'`. 

```javascript
function shortCirA(name){
  return name ??? 'John Doe';
}

function shortCirB(name){
  return name ??? 'Jane Doe';
}
```

- ||
- ??
- &&
- ==
````

Author: Ya I like that a lot -- Will add
